requirements_for_test.txt

```diff
@@ -1,9 +1,8 @@
 -r requirements.txt
 black==22.8.0  # Also update `.pre-commit-config.yaml` if this changes
-flake8==5.0.4  # Also update `.pre-commit-config.yaml` if this changes
-flake8-bugbear==22.9.23
+ruff==0.0.256  # Also update `.pre-commit-config.yaml` if this changes
```

Comment:
any reason you added 0.0.256 vs a more recent version? (i notice that they’re up to 261)

Reply:
🥲 .256 is so much nicer

But no, I think 256 was the latest version when I made the branch a few weeks ago and hadn't bumped it. But will now.